src/AppInstallerRepositoryCore/Rest/HttpClientHelper.cpp

```diff
@@ -6,32 +6,27 @@
 #include "cpprest/json.h"
 #include "HttpClientHelper.h"
 
-using namespace web;

```

Comment:
You can leave using namespace in the cpp files, it is the headers where it can lead to problems (well, it can lead to problems in the cpp but they will be contained to the cpp).

Reply:
Yeah, I misread that comment initially, by the time I read it again, I had removed the namespaces here. I will use them in cpp  files in the future.